docs/source/testing-and-issue-tracking/Kokkos-Project-Planning.md

```diff
@@ -0,0 +1,210 @@
+# Kokkos Project Planning
+
+## Requirements Gathering
+
+There are three requirement categories for the Kokkos Core project:
```

Comment:
this says three but below there are 4 bullets

Reply:
```suggestion
There are four requirement categories for the Kokkos Core project:
```